integration-tests/quarkus-sample/README.md

```diff
@@ -0,0 +1,118 @@
+# Quarkus sample
+
+This is a Quarkus sample about implementing REST endpoints using the Custom ConfigSource to get the secrets stored in
+Azure Key Vault.
+
+## Prerequisites
+
+To successfully run this sample, you need:
+
+* JDK 11+ installed with JAVA_HOME configured appropriately
+* Apache Maven 3.8.6+
+* Azure CLI and Azure subscription
+
+You also need to make sure the right version of dependencies are installed.
+
+### Use development iteration version
+
+By default, the sample depends on the development iteration version. To install the development iteration version, you
+need to build it locally.
+
+```
+# Switch to the root directory of this repository.
+# For example, if you are in the directory of azure-microprofile/integration-tests/quarkus-sample
+cd ../..
+
+# Install all Azure Extensions for MicroProfile locally.
+mvn clean install -DskipTests
+
+# Switch back to the directory of integration-tests/quarkus-sample
+cd integration-tests/quarkus-sample
+```
+
+## Preparing the Azure services
+
+The Custom ConfigSource implemented by `microprofile-config-keyvault` extension needs to connect to a real Azure Key Vault instance, follow steps below to create one.
+
+### Logging into Azure
+
+Sign in to Azure and create a resource group for hosting different Azure services to be created.
+
+```
+az login
+
+RESOURCE_GROUP_NAME=<resource-group-name>
+az group create \
+    --name ${RESOURCE_GROUP_NAME} \
+    --location eastus
+```
+
+### Creating Azure Key Vault
+
+Run the following commands to create an Azure Key Vault instance, add a few secrets, and export its uri as environment variables.
+
+```
+KEY_VAULT_NAME=<unique-key-vault-name>
+az keyvault create \
+    --name "${KEY_VAULT_NAME}" \
+    --resource-group "${RESOURCE_GROUP_NAME}" \
+    --location eastus
+
+az keyvault secret set \
+    --vault-name "${KEY_VAULT_NAME}" \
+    --name secret \
+    --value 1234
+az keyvault secret set \
+    --vault-name "${KEY_VAULT_NAME}" \
+    --name anotherSecret \
+    --value 5678
+
+export AZURE_KEYVAULT_URL=$(az keyvault show \
+  --resource-group "${RESOURCE_GROUP_NAME}" \
+  --name "${KEY_VAULT_NAME}" \
+  --query properties.vaultUri -o tsv)
+echo $AZURE_KEYVAULT_URL
+```
+
+Edit the file `src/main/resources/META-INF/microprofile-config.properties`. Uncomment all the lines. Make it so the value of the config property `azure.keyvault.url` is the value of `$AZURE_KEYVAULT_URL`.
+
+## Running the sample
+
+Now you can launch the sample with:
+
+```
+mvn package quarkus:run
+```
+
```

Comment:
I see this error on in the quarkus output. If this is expected, we should inform the reader that it is expected.

```
[ERROR] WorkloadIdentityCredential authentication unavailable. The workload options are not fully configured. See the troubleshooting guide for more information. https://aka.ms/azsdk/java/identity/workloadidentitycredential/troubleshoot
```

Reply:
Yes, this is expected. Will add some description in README to inform the reader that it is expected.

Reply:
Addressed in [c261780](https://github.com/Azure/azure-microprofile/pull/7/commits/c261780ae733007f513c4226ecef7abcf55c58a7).